### DISCUSSION TITLE
Linux Complete construction steps（求linux完整搭建过程，centos版本）

### DISCUSSION THREAD

**Discussion by SIZzzzzzzz** [Melledy/Grasscutter#2017] [Q&A]
Complete construction steps（求linux完整搭建过程，centos版本）
If yes, please send it to the mailbox（如果有请发送到邮箱）
1773232143@qq.com
Thank you.

Maybe we can discuss it here

**Comment by SIZzzzzzzz** [comment]
Server building process

**Comment by Musicminion** [comment]
欢迎前往网站：https://mihoyo-is-in.icu/
里面有详细的教程hhh
如果没有什么linux基础就还是别碰了（个人比较喜欢ubuntu，比较稳定

**Comment by Mrmanbanpai** [comment]
大概需要什么样的资源来部署此服务？
